microservices/nris_api/backend/app/helper.py

```diff
@@ -0,0 +1,13 @@
+from flask import url_for
+from flask_restplus import Api as BaseApi
+
+
+class Api(BaseApi):
+    """
+    Monkey patch Swagger API to return HTTPS URLs
+    """
+
+    @property
+    def specs_url(self):
+        scheme = 'http' if '5500' in self.base_url else 'https'
```

Comment:
Is this the same solution we're using in CORE? Seems like this would be configurable directly in Swagger

Reply:
Yup, same monkey patch in CORE, i didn't look it into it.

Reply:
yeah I had to monkey patch this in core mostly because of how swagger is built without reverse proxying in mind :(